plugins/grpc/handlers.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017, OpenCensus Authors
```

Comment:
Some files are `Copyright 2017 Google Inc.`...

Reply:
Good catch. Thank you. Fixed now.